README.Rmd

```diff
@@ -32,6 +38,91 @@ You can install the development version of ghqc from [GitHub](https://github.com
+
+- **Prepended Note**: A note to prepend to all checklists named "note" *(optional)*
+
+- **Logo**: A logo to be included in the header of the resultant record's header named as "logo.png"
+
+
+An example structure of the configuration information repository can be found at [github.com/A2-ai/ghqc.example_info_repo](https://github.com/A2-ai/ghqc.example_info_repo)
+
+## ghqc Setup
+
+ghqc has 3 main requirements that must be met before running any of the ghqc ecosystem apps:
+
+1. An environmental variable, `GHQC_INFO_REPO`, must be set to the configuration information repository url in "~/.Renviron". 
+
+2. The configuration information repository must be downloaded to a local location. 
+
+3. All of the dependency packages for the ghqc shiny apps, including `ghqc.app` must be located in the same directory. 
+
+Within this package, a function `ghqc::setup_ghqc()` aids in this setup.
+
+``` r
+ghqc::setup_ghqc()
+```
+
+1. **Renviron Setup**
+
+Enter the url of the configuration information repository described above
+
+![](man/figures/renviron_setup.png)
+
+2. **Configuration Information Download and Verification**
+
+The configuration information will default to installing to "~/.local/share/ghqc/<repo_name>". To download to the default location, which is highly recommended, hit `ENTER`. Otherwise, type in the path in which to download the repository.
+
+In addition to downloading the repository, or remote updates if the repository was previously downloaded, the structure of the repository and the checklist yamls are verified.
+
+![](man/figures/info_repo_download.png)
+
+3. **Shiny App Dependency Installation**
+
+Select whether to install the dependency packages or symlink to previously a previously installed package library. 
+
+- INSTALL PACKAGES
+
+  - NOTE: The install option is currently only available to linux systems supported by Posit Package Manager (PPM) and will install from [PPM@2024-03-01](https://packagemanager.posit.co/cran/2024-03-01). 
+  
+  - The dependency packages are installed by default to "~/.local/share/ghqc/rpkgs". To install to the default location, which is highly recommended, hit `ENTER`. Otherwise, type in the path in which to install the dependency packages to.
+  
+  ![](man/figures/install_depends.png)
+
+- SYMLINK PACKAGES  
+  
+  - NOTE: The link option requires all packages be installed in the same `link_path` AND they meet the requirements listed in the imports
+  
+  - Provide the path to the previously installed package library
+  
+  - The dependency packages are symlinked by default to "~/.local/share/ghqc/rpkgs". To symlink to the default location, which is highly recommended, hit `ENTER`. Otherwise, type in the path in which to download the repository.
+  
+  ![](man/figures/link_depends.png)
+
+#### ghqc.app installation
+
+You can install the development version of ghqc.app from [GitHub](https://github.com/) with:
```

Comment:
Maybe a special note that the only part missing from the setup is this

Reply:
https://github.com/A2-ai/ghqc/blob/b22c559d5c7ec223217c29fb4be56437a933fce8/README.Rmd#L118

Done!